src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -133,8 +132,26 @@ public void openBrowserAndConnect(String uri) {
     openBrowserAndConnect(uri, null);
   }
 
-  public void openBrowserAndConnect(String uri, String page) {
-    openBrowserImpl(uri, page);
+  public void openBrowserAndConnect(String uri, String screen) {
+    openBrowserImpl(uri, screen);
+  }
+
+  /**
+   * Open a browser connected to DevTools with the given (optional) screen.
+   * <p>
+   * Calling this method may cause DevTools to be installed (in which case, this API will provide
+   * appropriate progress feedback to the user).
+   */
+  public void openToScreen(@NotNull FlutterApp app, @Nullable String screen) {
```

Comment:
As long as this behaves properly when screen is null, lgtm

Reply:
Yup, it should (it's existing logic in DevToolsUtils.generateDevToolsUrl()).